client-sdk/ts-web/core/reflect-go/main.go

```diff
@@ -563,6 +566,8 @@ var skipMethods = map[string]bool{
 	"github.com/oasisprotocol/oasis-core/go/consensus/api.ClientBackend.Governance": true,
 	"github.com/oasisprotocol/oasis-core/go/consensus/api.ClientBackend.RootHash":   true,
 	"github.com/oasisprotocol/oasis-core/go/consensus/api.ClientBackend.State":      true,
+	"github.com/oasisprotocol/oasis-core/go/keymanager/api.Backend.Secrets":         true,
+	"github.com/oasisprotocol/oasis-core/go/keymanager/api.Backend.Churp":           true,
```

Comment:
will need some extra visitClientWithService calls for https://github.com/oasisprotocol/oasis-core/blob/master/go/keymanager/churp/grpc.go#L15 and https://github.com/oasisprotocol/oasis-core/blob/master/go/keymanager/churp/grpc.go#L15